src/renderer/App.css

```diff
@@ -71,6 +71,11 @@ div#root {
   padding: 0;
 }
 
+.MuiFormControlLabel-root .MuiFormControlLabel-label.Mui-disabled,
+.Mui-disabled .MuiCheckbox-root .MuiSvgIcon-root {
+  color: #666 !important;
```

Comment:
I had to put `!important` on the CSS rule as I could not for the life of me figure out where the styling was set and I couldn't get more specific with the rule I made. Suggestions are welcome.